pkgs/development/python-modules/blobfile/default.nix

```
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchPypi
+, setuptools
+, urllib3
+, pycryptodomex
+, lxml
+, filelock
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "blobfile";
+  version = "2.0.1";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+
+    hash = "sha256-b1Gz6UjzCpLnNKl0sk/ND2pRhB/Qg96WiJkjFIE1jaI=";
+  };
+
+  propagatedBuildInputs = [
+    urllib3
+    pycryptodomex
+    lxml
+    filelock
+  ];
+
+
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "blobfile"
+  ];
+
+  meta = with lib; {
+    description = "This is a library that provides a Python-like interface for reading local and remote files from blob storage";
+    license = licenses.mit;
```

```suggestion
    license = licenses.unlicense;
```

The upstream repo uses unlicense rather than MIT (see [here](https://github.com/christopher-hesse/blobfile/blob/d099063d0f732eddb63aa8a22dcfab4d605a99ea/LICENSE))